R/DMMParam.R

```diff
@@ -1,29 +1,30 @@
 #' DMM clustering
 #' 
-#' Run the \code{DMM} function on a distance matrix within \code{\link{clusterRows}}.
+#' Run the \code{DMM} function on a distance matrix 
+#' within \code{\link{clusterRows}}.
 #' 
-#' @param k An integer indicating the number of clusters, or a list of integers defining the possible number of clusters to choose from.
-#' @param type A string specifying the fit function to use to find the optimal number of clusters. Use in combination with k being a list of integers
-#' @param type A boolean specifying 
-#' @param seed An integer specifying the seed to use when doing the DMM algorithm. Will be random if not set.
+#' @param k An integer indicating the number of clusters, or a list of integers 
+#'   defining the possible number of clusters to choose from.
+#' @param type A string specifying the fit function to use to find the optimal 
+#'   number of clusters. Use in combination with k being a list of integers. 
+#'   Must be equal to \code{laplace}, \code{AIC} or \code{BIC}.
+#' @param transposed Logical scalar, is x transposed with samples in rows?
```

Comment:
Do we need this if there is MARGIN?

Reply:
Well there is margin in the mia wrapper. But this is (for now), in the bluster package. So people using it don't necessarily have access to the mia wrapper.
If we include the DMM-class in mia I guess we could remove the `transposed` parameter but if it stays separate from mia, I think we should leave it that way.

Reply:
I see. This is a new method in bluster? Did we already get approval for adding DMM?

Reply:
Well it was one of the actions points in slack discussion. Following our discussion, I needed to do a bluster-like method to see if it's more interesting than the current DMM in mia.
I took the liberty to do it in a bluster branch because it was more convenient. 

I guess now the next step is to decide whether we put this in mia as decided, or in bluster, or if we keep the original DMM in mia.

Sorry, I wasn't clear the first time. 
I can add this in a new mia branch if you prefer.

Reply:
Yes, this is good. In any case it is better to have this cluster method as a bluster-like method (if not in bluster, in mia)

--> let's ask Aaron